### PR TITLE
Update to lodash 4

### LINKED ISSUE #283: _.isNil doesn't exist in lodash 3.10, which breaks config.txt

### PR DESCRIPTION
**Pull request by Page-**
Fixes #283

### CODE REVIEW

**Comment by pcarranzav** [comment]
Fixes #283

**Comment by Page-** [comment]
Yeah, tests are definitely needed, the way I checked was to go through the lodash changelog and searched for things list there in the code, and also went through the uses of lodash in the code and searched for changes to those uses in the lodash changelog, so it got checked in both ways to be a bit safer (I've found this method works pretty well for other places I've converted, but it's always possible that something could slip through)

File: src/proxyvisor.coffee [hunk 1]
```diff
@@ -218,9 +218,9 @@ exports.fetchAndSetTargetsForDependentApps = (state, fetchFn, apiKey) ->
 		localApps = _.indexBy(localDependentApps, 'appId')
```
Comment (pcarranzav) [inline src/proxyvisor.coffee]: @Page- this needs to be replaced by `_.keyBy(localDependentApps, 'appId')`

Comment (Page-) [inline src/proxyvisor.coffee]: Fixed, I think I might have missed it in a merge

**Comment by pcarranzav** [comment]
I've given this a bit of testing and it seems okay. I recommend some testing on your side to make sure, but looks good to merge.

**Pull request merged**